lib/offsite_payments/integrations/realex_offsite.rb

```diff
@@ -213,6 +503,48 @@ def copy_billing_address
                    add_field("HPP_SHIPPING_#{k.split('HPP_BILLING_')[1]}", v)
                  end
         end
+
+        # Validations
+        def get_pattern(key)
+          return case key
```

Comment:
Here you don't need the `return` keyword, since it is the last expression being evaluated, `return` is used when you want to break/leave a block/logic earlier.

```suggestion
          case key
```

Reply:
Updated both of them.